packages/@react-spectrum/card/stories/Card.stories.tsx

```diff
@@ -10,186 +10,38 @@
 
-import assetStyles from '@adobe/spectrum-css-temp/components/asset/vars.css';
-import {Avatar} from '@react-spectrum/avatar';
 import {Card} from '..';
-import {CardBase} from '../src/CardBase';
-import {CardViewContext} from '../src/CardViewContext';
-import {classNames, useSlotProps, useStyleProps} from '@react-spectrum/utils';
+import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
 import {Content} from '@react-spectrum/view';
+import {Default} from '../chromatic/Card.chromatic';
 import {getDescription, getImage} from './utils';
 import {Heading, Text} from '@react-spectrum/text';
-import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
 import {Image} from '@react-spectrum/image';
-import {Meta, Story} from '@storybook/react';
-import React, {Dispatch, SetStateAction, useState} from 'react';
+import React from 'react';
 import {SpectrumCardProps} from '@react-types/card';
-import {usePress} from '@react-aria/interactions';
 import {useProvider} from '@react-spectrum/provider';
 
-// see https://github.com/storybookjs/storybook/issues/8426#issuecomment-669021940
-const StoryFn = ({storyFn}) => storyFn();
-
-const meta: Meta<SpectrumCardProps> = {
+export default {
   title: 'Card/default',
-  component: Card,
-  decorators: [storyFn => <StoryFn storyFn={storyFn} />]
-};
-
-export default meta;
-
-
-const Template = (): Story<SpectrumCardProps> => (args) => (
-  <div style={{width: '208px'}}>
-    <Card {...args} />
-  </div>
-);
-
-/* This is a bit of a funny template, we can't get selected on a Card through context because
-* if there's context it assumes it's being rendered in a collection. It's just here for a quick check of styles. */
-interface ISelectableCard {
-  disabledKeys: Set<any>,
-  selectionManager: {
-    isSelected: () => boolean,
-    select: () => Dispatch<SetStateAction<ISelectableCard>>
-  }
-}
-let SelectableCard = (props) => {
-  let [state, setState] = useState<ISelectableCard>({
-    disabledKeys: new Set(),
-    selectionManager: {
-      isSelected: () => true,
-      select: () => setState(prev => ({
-        ...prev,
-        selectionManager: {
-          ...prev.selectionManager,
-          isSelected: () => !prev.selectionManager.isSelected()
-        }
-      }))
-    }
-  });
-  let {pressProps} = usePress({onPress: () => setState(prev => ({
-    ...prev,
-    selectionManager: {
-      ...prev.selectionManager,
-      isSelected: () => !prev.selectionManager.isSelected()
-    }
-  }))});
-  return (
-    <div style={{width: '208px'}} {...pressProps}>
-      <CardViewContext.Provider value={{state}}>
-        <CardBase {...props} />
-      </CardViewContext.Provider>
-    </div>
-  );
+  component: Card
+} as ComponentMeta<typeof Card>;
+
+export type CardStory = ComponentStoryObj<typeof Card>;
+
+export const CardGrid: CardStory = {
+  render: (args, context) => <Card {...args} {...context} />,
+  args: {layout: 'grid'},
+  decorators: [(Story, context) => (
+    <Decorator
```

Comment:
Decorator is a component that uses Provider Context, and since decorators aren't React components, they can't use the context directly
You'll notice another story further down where I didn't do this because it didn't use context
Do we generally like this style of using the decorator to do most of the lifting or the other style in here of creating a custom render to handle everything

Reply:
yep, so I could make the component that I use as the decorator more complex to be shared across all of them, something with a signature like 
```
    <Decorator
      Story={Story}
      context={context}
      wrapperProps={{style: {
          width: '100%',
          margin: '50px',
          display: 'grid',
          gap: '20px',
          gridTemplateColumns: 'repeat(auto-fit, 208px)',
          gridAutoRows: 'auto',
          justifyContent: 'center',
          justifyItems: 'center',
          alignItems: 'start'
        }}}
      cardWrapperProps={{medium: {style: {width: '208px', height: '293px'}}, large: {style: {width: '208px', height: '355px'}}}}
      exampleSettings={{showImage: true, headingText: 'Title', detailText: 'PNG', contentText: 'Description'}} />
```

Reply:
Gotcha, I'm fine with that. Perhaps we just use best judgement on whether to use this Decorator style or the custom render style then.

Reply:
Resolving with: tried combining decorators, there are just enough differences that the shared decorator approach makes things harder instead of easier
instead, I've opted for each of these to have it's own decorator so I can still standardize all stories with the same types in their signatures. this still makes it easy to add or modify individual stories